comps/comps-foreman-rhel7.xml

```diff
@@ -326,11 +326,11 @@
       <packagereq type="default">rubygem-rb-inotify</packagereq>
       <packagereq type="default">rubygem-rsec</packagereq>
       <packagereq type="default">rubygem-rubyipmi</packagereq>
+      <packagereq type="default">nodejs-jquery-doc</packagereq>
```

Comment:
@apuntamb I am sorry, your original diff was actually correct. the `nodejs-jquery-doc` package was removed in 37cd622924b34a5033baee81791829f9927329b4, so you can safely remove this

Reply:
done. :)